src/main/java/tc/oc/pgm/api/player/MatchPlayer.java

```diff
@@ -138,6 +138,14 @@
    */
   boolean isFrozen();
 
+  /**
+   * Get whether the {@link MatchPlayer} us currently muted. Muted players can not send commands to
+   * global or team chat.
+   *
+   * @return Whether the {@link MatchPlayer} is muted.
+   */
+  boolean isMuted();
```

Comment:
Not sure I want to bring this state into `MatchPlayer`. I would rather `ChatDispatcher` store this information in a `Set<UUID> muted`.

Reply:
Alrighty! Will make that change asap.